exporting/prometheus/prometheus.c

```diff
@@ -526,6 +526,83 @@ static void generate_as_collected_prom_metric(BUFFER *wb,
         buffer_sprintf(wb, "\n");
 }
 
+static void prometheus_print_os_info(
+    BUFFER *wb,
+    PROMETHEUS_OUTPUT_OPTIONS output_options)
+{
+    FILE *fp = fopen("/etc/os-release", "r");
```

Comment:
We need to use `netdata_configured_host_prefix` here. Like:

https://github.com/netdata/netdata/blob/fbe5a35a8d00976febf10733bb1b0506b23d9828/collectors/cgroups.plugin/sys_fs_cgroup.c#L186-L187


Otherwise, the OS information will be the OS information of the container in docker.

Reply:
Thanks! Addressed in last commit.